Unix/http/httpclientauth.c

```diff
@@ -227,12 +435,20 @@ _GssUnloadLibrary()
     _g_gssClientState.gssSetNegMechs = NULL;
     _g_gssClientState.gssAcquireCredwithPassword = NULL;
     _g_gssClientState.gssLibLoaded = NOT_LOADED;
+
+    if (_g_gssClientState.credFileName)
+    {
+        unlink(_g_gssClientState.credFileName);
```

Comment:
No error checking on a system routine. Do you want one?

And, I'm not quite sure where this gets set (I don't see it set as part of this changeset). Will it only be deleted if the environment variable doesn't point to one?

What if the environment variable is defined but points to something that does not exist?